Reference/using-ioc.md

```diff
@@ -11,6 +11,20 @@ We don't use IoC in the Umbraco source code. This isn't because we don't like it
 In most IoC frameworks you would setup your container in your global.asax class. To do that in Umbraco, you will need to inherit from our global.asax class called: `Umbraco.Web.UmbracoApplication`. You should then override the `OnApplicationStarted` method to build your container and initialize any of the IoC stuff that you require.
 Alternatively you can implement the `Umbraco.Web.IApplicationEventHandler` interface.
 
+## What assemblies and controllers do I need to register?
+
+You need to register all assemblies that may contain MVC or WebApi controllers. In Umbraco this is the `umbraco` assembly which you can get a direct assembly reference to using the example syntax used above:
```

Comment:
```suggestion
You need to register all assemblies that may contain MVC or WebApi controllers. In Umbraco this is the `umbraco` assembly which you can get a direct assembly reference to using the example syntax used below:
```
It's below, right? 🙂

Reply:
@jmayntzhusen yes i think so :) // @mzajkowski